apps/teams-test-app/src/App.tsx

```diff
@@ -68,12 +68,15 @@ import WebStorageAPIs from './components/WebStorageAPIs';
 
 const urlParams = new URLSearchParams(window.location.search);
 
+const getOriginParam = urlParams.has('origins') && urlParams.get('origins') ? urlParams.get('origins') : '';
+const validMessageOrigins: string[] = getOriginParam ? [getOriginParam] : [];
+
 // This is added for custom initialization when app can be initialized based upon a trigger/click.
 if (!urlParams.has('customInit') || !urlParams.get('customInit')) {
   if (isTestBackCompat()) {
-    initialize();
+    initialize(undefined, validMessageOrigins);
   } else {
-    app.initialize();
+    app.initialize(validMessageOrigins);
```

Comment:
Should it be passing empty array or undefined if `getOriginParam` is not specified? Maintaining the previous behavior if it's not being overridden would be passing undefined.

Reply:
The check on line 72 ensures we pass in the correct `validMessageOrigins`